src/lib.rs

```diff
@@ -230,19 +263,42 @@ impl Contract {
         notify::notify_mentions(desc.as_str(), id)
     }
 
+    // Usage without accepted_terms_and_conditions_version is deprecated. 
+    // So accepted_terms_and_conditions_version should be used 
+    // as it will become mandatory in the future
     #[payable]
     pub fn add_proposal(
         &mut self,
         body: VersionedProposalBody,
         labels: HashSet<String>,
+        accepted_terms_and_conditions_version: Option<near_sdk::BlockHeight>,
```

Comment:
I guess we need to use Option for now as otherwise it will break frontend immediately. What do you think?

Reply:
Let's release a UI that will pass this argument first. Unknown fields that are passed from the UI are ignored, so if we release the UI first, it won't break and then we will release the contract and it will work as expected.

Reply:
Created a separate draft PR for that: https://github.com/NEAR-DevHub/neardevhub-contract/pull/117